contracts/PodCastNFT.sol

```diff
@@ -13,15 +13,19 @@ contract PodCastNFT is ERC721URIStorage, Ownable {
     Counters.Counter private _tokenIds;
     IERC20 public henkakuToken;
     uint256 public price;
+    address public fundAddress;
 
     mapping(address => string[]) public roles;
     mapping(uint256 => bool) private _communityMemberShip;
 
     event BoughtMemberShipNFT(address _owner, uint256 _amount);
 
-    constructor(address _erc20) ERC721("Henkaku v0.2", "henkaku") {
+    constructor(address _erc20, address _fundAddress)
+        ERC721("Henkaku v0.2", "henkaku")
+    {
         henkakuToken = IERC20(_erc20);
         setPrice(1000e18);
+        fundAddress = _fundAddress;
```

Comment:
It's crazy detailed, but I felt that if you were going to setPrice, you could align it with setFundaddress. You can also assign both.

Reply:
thanks! true I usually forget like this things